### PR TITLE
Added simple name spaces to ekg-core registerGcMetrics.

### PR DESCRIPTION
**Pull request by smurphy8**
Just a small change so you can write:
``` haskell
registerNamedGcMetrics "myapp" store
-- produces: 'myapp.rts.gc.xxx' for all Store entries
```

### CODE REVIEW

File: System/Metrics.hs [hunk 1]
```diff
@@ -407,32 +409,44 @@ toMs s = round (s * 1000.0)
 -- @par_tot_bytes_copied@ divided by @par_max_bytes_copied@ approaches
 -- 1 for a maximally sequential run and approaches the number of
 -- threads (set by the RTS flag @-N@) for a maximally parallel run.
+
 registerGcMetrics :: Store -> IO ()
-registerGcMetrics store =
+registerGcMetrics  = registerNamedGcMetrics ""
+
+
+-- | same as registerGcMetrics but you can append a namespace
```
Comment (tibbe) [inline System/Metrics.hs]: Start sentence with capital letter.
Surround identifiers with single quotes to make them links.

File: System/Metrics.hs [hunk 2]
```diff
@@ -407,32 +409,44 @@ toMs s = round (s * 1000.0)
...
+-- >>> registerNamedGcMetrics "myapp" store
+-- produces: 'myapp.rts.gc.xxx' for all Store entries
```
Comment (tibbe) [inline System/Metrics.hs]: Produces

File: System/Metrics.hs [hunk 3]
```diff
@@ -1,8 +1,8 @@
-{-# LANGUAGE BangPatterns #-}
-{-# LANGUAGE CPP #-}
+{-# LANGUAGE BangPatterns              #-}
```
Comment (tibbe) [inline System/Metrics.hs]: Please don't reformat other people's code to suite your personal preference. :)

File: ekg-core.cabal [hunk 4]
```diff
@@ -1,5 +1,5 @@
 name:                ekg-core
-version:             0.1.1.0
+version:             0.1.1.1
```
Comment (tibbe) [inline ekg-core.cabal]: You need to bump the minor version (i.e. 3rd component) instead.

Comment (smurphy8) [inline ekg-core.cabal]: Sorry I forgot I had stylish on :(

**Pull request closed without merging**